File: windows/src/GoalMaker.Core/Composer/SpanKind.cs
namespace GoalMaker.Core.Composer;

/// <summary>What a recognized part of a composer line is (docs/composer.md).</summary>
public enum SpanKind
{
    Date,
    Time,
    Repeat,
    Tag,
    Area,
    Project,
    Priority,
    Idea,
    Command,
}

File: android/app/src/main/kotlin/com/goalmaker/app/domain/composer/SpanKind.kt
package com.goalmaker.app.domain.composer

/** What a recognized part of a composer line is (docs/composer.md). */
enum class SpanKind {
    DATE,
    TIME,
    REPEAT,
    TAG,
    AREA,
    PROJECT,
    PRIORITY,
    IDEA,
    COMMAND,
}
